plugins/woocommerce-blocks/assets/js/data/cart/default-state.ts

```diff
@@ -57,6 +59,7 @@ Object.keys( defaultFields ).forEach( ( key ) => {
 export const defaultCartState: CartState = {
 	cartItemsPendingQuantity: EMPTY_PENDING_QUANTITY,
 	cartItemsPendingDelete: EMPTY_PENDING_DELETE,
+	productsPendingAdd: EMPTY_PENDING_ADD,
```

Comment:
Again, differentiating between _cart items_ (what products become when they are _in_ the cart) vs. _products_ (a product in the store that shoppers add to their cart)